src/attributes-to-props.ts

```diff
@@ -69,12 +69,12 @@ export default function attributesToProps(
 
       // convert attribute to uncontrolled component prop (e.g., `value` to `defaultValue`)
       if (
-        UNCONTROLLED_COMPONENT_ATTRIBUTES.indexOf(
+        UNCONTROLLED_COMPONENT_ATTRIBUTES.includes(
           propName as UncontrolledComponentAttributes,
-        ) !== -1 &&
-        UNCONTROLLED_COMPONENT_NAMES.indexOf(
+        ) &&
+        UNCONTROLLED_COMPONENT_NAMES.includes(
           nodeName! as UncontrolledComponentNames,
-        ) !== -1 &&
+        ) &&
         !isInputValueOnly
       ) {
```

Comment:
I understood the code above to be checking whether a particular string is included or not, so I thought the `includes` method, which returns a `Boolean`, was more intuitive.

But obviously this is a matter of taste, so your opinion as a maintainer is even more important!

Reply:
The reason `indexOf` was used before was to support older browsers. I think we should be safe to use `includes` now. What do you think? https://caniuse.com/?search=includes